packages/react/src/themes/teams/components/Label/labelStyles.ts

```diff
@@ -2,17 +2,20 @@ import { pxToRem } from '../../../../lib'
 import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { LabelProps } from '../../../../components/Label/Label'
 import { LabelVariables } from './labelVariables'
+import { getColorScheme } from '../../colors'
 
 const labelStyles: ComponentSlotStylesInput<LabelProps, LabelVariables> = {
-  root: ({ props: p, variables: v, colors }): ICSSInJSStyle => {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const colors = getColorScheme(v.colorScheme, p.color)
```

Comment:
nit: result of `getColorScheme()` is stored to `colors` in 3 files to `colorScheme` in another 4. Let's choose one name and stick with that.

Reply:
The `colors` as a name is used everywhere now. It looks more natural to use `colors.foreground`, `colors.backgroundDefault` etc.